packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -3219,6 +3219,23 @@
       }
     }
   },
+  "security:reference-data": {
+    "dynamic": false,
+    "properties": {
+      "id": {
+        "type": "keyword"
+      },
+      "metadata": {
```

Comment:
are these mappings for properties that are being searched over? With `dynamic: false` there's no need to specify the mappings if you just want to store the data.

https://github.com/elastic/kibana/blob/main/dev_docs/tutorials/saved_objects.mdx?plain=1#L125-L127

Reply:
Currently, no we are not searching on it.
Any negative side affect by having it defined this way?

Reply:
Our recommendation is to not add fields to the mappings unless they need to be indexed / searched on:
* This way we promote slower growth of the whole set of mappings for the SO index.
* We have [schema](https://github.com/elastic/kibana/blob/75c0f9543acc80e0d875e908f8c4570581393e20/src/core/packages/saved-objects/server/src/saved_objects_type.ts#L97) validation that helps enforcing type constraints.
* In case you want to update any of these fields in the future, it gives you a certain degree of flexibility (NB once you define something in the mappings it's kinda set in stone, as we don't allow breaking changes to the mappings).